Hm, I'm running `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"` locally and I get a failed build for webkitgtk (which seems to be a dependency for lutris), but I don't see how this would have anything to do with my changes?

```
error: --- Error --- nix-daemon
builder for '/nix/store/cd9napir1g28rx44zimyxjwch6k52wca-webkitgtk-2.28.4.drv' failed with exit code 1; last 10 log lines:
  In file included from DerivedSources/WebCore/unified-sources/UnifiedSource-44a97133-1.cpp:4:
  ../Source/WebCore/crypto/SubtleCrypto.cpp: In member function 'void WebCore::SubtleCrypto::deriveKey(JSC::JSGlobalObject&, WebCore::SubtleCrypto::AlgorithmIdentifier&&, WebCore::CryptoKey&, WebCore::SubtleCrypto::AlgorithmIdentifier&&, bool, WTF::Vector<WebCore::CryptoKeyUsage>&&, WTF::Ref<WebCore::DeferredPromise>&&)':
  ../Source/WebCore/crypto/SubtleCrypto.cpp:769:6: internal compiler error: Segmentation fault
    769 | void SubtleCrypto::deriveKey(JSC::JSGlobalObject& state, AlgorithmIdentifier&& algorithmIdentifier, CryptoKey& baseKey, AlgorithmIdentifier&& derivedKeyType, bool extractable, Vector<CryptoKeyUsage>&& keyUsages, Ref<DeferredPromise>&& promise)
        |      ^~~~~~~~~~~~
  Please submit a full bug report,
  with preprocessed source if appropriate.
  See <https://gcc.gnu.org/bugs/> for instructions.
  [3747/4503] Building CXX object Source/WebCore/CMakeFiles/WebCore.dir/__/__/DerivedSources/WebCore/unified-sources/UnifiedSource-2f84417a-12.cpp.o
  ninja: build stopped: subcommand failed.
```

Similarly

```
error: --- Error --- nix-daemon
builder for '/nix/store/mrdiyznpbsnxwypwrlxw54rk5aqasxv4-vstsdk368_08_11_2017_build_121.zip.drv' failed with exit code 1; last 10 log lines:
  ***
  Unfortunately, we cannot download file vstsdk368_08_11_2017_build_121.zip automatically.
  Please go to http://www.steinberg.net/en/company/developers.html to download it yourself, and add it to the Nix store
  using either
    nix-store --add-fixed sha256 vstsdk368_08_11_2017_build_121.zip
  or
    nix-prefetch-url --type sha256 file:///path/to/vstsdk368_08_11_2017_build_121.zip
  
  ***
```

--- a/pkgs/misc/emulators/wine/sources.nix
+++ b/pkgs/misc/emulators/wine/sources.nix
@@ -39,16 +39,16 @@ in rec {
 
   unstable = fetchurl rec {
     # NOTE: Don't forget to change the SHA256 for staging as well.
-    version = "5.16";
+    version = "5.17";
     url = "https://dl.winehq.org/wine/source/5.x/wine-${version}.tar.xz";
-    sha256 = "0j9268s1dy4cjvhcf4igbg54gaws4a1l3pda449qy2p2i4psdncq";
+    sha256 = "0kqy184x16g1lfdyc46dj3cbs4zl0j66w2hb04qq30ql2g88nhzn";
     inherit (stable) mono gecko32 gecko64;
   };
 
   staging = fetchFromGitHub rec {
     # https://github.com/wine-staging/wine-staging/releases
     inherit (unstable) version;
-    sha256 = "1rrw15mrygv9zcbqz0c3s7q7971wqj89ys2bvvm4b0d2h4j0k6wq";
+    sha256 = "1mbsglv7154wlv2ryr8jsvvwjawzv8fl6rj1kqrl3zidczl6a2aj";
     owner = "wine-staging";
     repo = "wine-staging";
     rev = "v${version}";

--- a/pkgs/misc/emulators/wine/sources.nix
+++ b/pkgs/misc/emulators/wine/sources.nix
@@ -13,9 +13,9 @@ let fetchurl = args@{url, sha256, ...}:
 in rec {
 
   stable = fetchurl rec {
-    version = "5.0";
+    version = "5.0.2";
     url = "https://dl.winehq.org/wine/source/5.0/wine-${version}.tar.xz";
-    sha256 = "1d0kcy338radq07hrnzcpc9lc9j2fvzjh37q673002x8d6x5058q";
+    sha256 = "1g7c5djbi262h6zivmxbk26rsflwq8mrghr7hci3ajw7f3s89hn2";
 
     ## see http://wiki.winehq.org/Gecko
     gecko32 = fetchurl rec {

--- a/pkgs/misc/emulators/wine/vkd3d.nix
+++ b/pkgs/misc/emulators/wine/vkd3d.nix
@@ -5,11 +5,11 @@
 
 stdenv.mkDerivation rec {
   pname = "vkd3d";
-  version = "1.1";
+  version = "1.2";
 
   src = fetchurl {
     url = "https://dl.winehq.org/vkd3d/source/vkd3d-${version}.tar.xz";
-    sha256 = "1dkayp95g1691w7n2yn1q9y7klq5xa921dgmn9a5vil0rihxqnj9";
+    sha256 = "0szr1lw3xbgi9qjm13d1q4gyzzwv8i5wfxiwjg6dmwphrc7h6jxh";
   };
 
   buildInputs = [ vulkan-headers spirv-headers vulkan-loader ];

